_data/sites.yml

```
@@ -2018,6 +2018,11 @@
   size: 243
   last_checked: 2022-04-26
 
+- domain: rohandebsarkar.github.io
+  url: https://rohandebsarkar.github.io/
+  size: 392
```

```suggestion
  size: 226
```
I just updated my site. New GTmetrix report: https://gtmetrix.com/reports/rohandebsarkar.github.io/B62Dlfcq/